library/lib/eval-in-fork.js

```diff
@@ -29,3 +29,9 @@ function attempt(f) {
     process.exit(1)
   }
 }
+
+function evalSource(source) {
+  const file = path.resolve(`.kaliber-eval`, `${uuid.v4()}.js`)
+  fs.outputFileSync(file, source, { encoding: 'utf-8' })
+  try { return require(file) } finally { fs.removeSync(file) }
```

Comment:
Waarom slaan we het bestand eerst op en doen we dan een require van het bestand?